cli/Valet/Site.php

```diff
@@ -220,6 +220,20 @@ function getSites($path, $certs)
         });
     }
 
+    /**
+     * Get Stub file for securing site
```

Comment:
Standard single-line comment syntax in this repo is to end the sentence with a period `.`